velox/docs/develop/debugging/print-plan-with-stats.rst

```diff
@@ -225,6 +266,13 @@ printPlanWithStats shows this information as “Blocked wall time”.
 
 	Blocked wall time: 10.00us
 
+Some operators like TableScan may produce the dynamic filter, reports the plan node ids.
```

Comment:
TableScan doesn't produce dynamic filters. These are produced by HashProbe operator. This doc update seems unrelated to the changes in the PR. Would you extract it in a separate PR so we can get it right without blocking this PR?

Reply:
Sure, changed, only left CPU Time and CPU breakdown change in this document.